docs/getting-started-with-sylius/shop-customizations.rst

```diff
@@ -30,7 +30,24 @@ Next, replace the ``img`` element source with a link to the logo or properly imp
 `Symfony assets documentation <https://symfony.com/doc/current/best_practices/web-assets.html>`_ for more info).
 
 The other way to achieve this is to modify the configuration of the ``sylius.shop.layout.header.grid`` template event.
-Learn more about it in :doc:`this doc</customization/template>`.
+Using the same logo file as from example above ``templates/bundles/SyliusShopBundle/Layout/Header/_logo.html.twig``.
```

Comment:
```suggestion
Using the same logo file as from the example above ``templates/shop/Layout/Header/_logo.html.twig``.
```

Reply:
@lchrusciel Your proposal does not seems to work.
Also I am using example from above (which is exact copy of line 28), so I don't know if this change at least here would be correct 🤔 

![image](https://user-images.githubusercontent.com/35863747/165562156-c793816f-6fe9-4bbc-86f7-11a9154a4bb9.png)

Reply:
you are correct, line 28 of the `docs/getting-started-with-sylius/shop-customizations.rst` file should be adjusted as well, where we should replace the `templates/bundles/SyliusShopBundle/Layout/Header/_logo.html.twig` file with `templates/shop/Layout/Header/_logo.html.twig`